scaleway/helpers_baremetal.go

```diff
@@ -171,3 +171,43 @@ func baremetalFindOfferByID(ctx context.Context, baremetalAPI *baremetal.API, zo
 
 	return nil, fmt.Errorf("offer %s not found in zone %s", offerID, zone)
 }
+
+func baremetalCompareOptionIDsToAdd(modifiedOptionIDs, currentOptionIDs []string, zone scw.Zone) []string {
```

Comment:
Is there a way to use a SetOption? Maybe it would be worth asking the team if they can add this feature

Reply:
This is will be resolved in other issue